src/core/server/saved_objects/migrations/actions/open_pit.ts

```diff
@@ -25,7 +25,7 @@ export interface OpenPitParams {
   index: string;
 }
 // how long ES should keep PIT alive
-export const pitKeepAlive = '10m';
+export const PIT_KEEP_ALIVE_10MINS = '10m';
```

Comment:
I would prefer not to include the value of this `const` in its name.
If tomorrow we want to change it to 11 mins we'll have to refactor/rename all usages.